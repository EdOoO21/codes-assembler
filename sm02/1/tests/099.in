-221 -591 227 -634 -529 -492 -728 556 -823 -486 493 -216 87 401 434 103 -817 921 277 2 -592 -137 479 255 205 -557 623 847 614 -235 684 -206 764 397 -565 -689 -55 352 -634 740 161 -307 -831 939 467 82 387 586 -821 -40 -353
