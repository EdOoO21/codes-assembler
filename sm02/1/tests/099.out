-353
-40
-821
586
387
82
467
939
-831
-307
161
740
-634
352
-55
-689
-565
397
764
-206
684
-235
614
847
623
-557
205
255
479
-137
-592
2
277
921
-817
103
434
401
87
-216
493
-486
-823
556
-728
-492
-529
-634
227
-591
-221
